src/main/java/frc/robot/autochooser/SwerveMotor.java

```diff
@@ -0,0 +1,112 @@
+package frc.robot.autochooser;
+
+import frc.robot.Constants2023;
+import frc.robot.Constants2024;
+
+public enum SwerveMotor {
+    
+    
+
+    MKI(false, false, false, false, 
+    Constants2024.CHASSIS_STEER_GEAR_RATIO, Constants2024.CHASSIS_DRIVE_GEAR_RATIO, 
```

Comment:
Copy the actual values from the Constants file - do not reference the Constants from here. Eventually we want to delete these constants from the Constants class.